.github/workflows/ci.yaml

```
@@ -157,6 +157,14 @@ jobs:
             os_version: 15.4
           - toxenv: base
             os_version: 15.7
+          - toxenv: all
+            os_version: 16.0
+          - toxenv: base
+            os_version: 16.0
+          - toxenv: minimal
+            os_version: 16.0
+          - toxenv: metadata
+            os_version: 16.0
```

Don't we all love YAML:
```suggestion
          - toxenv: all
            os_version: "16.0"
          - toxenv: base
            os_version: "16.0"
          - toxenv: minimal
            os_version: "16.0"
          - toxenv: metadata
            os_version: "16.0"
```